.github/workflows/publish.yml

```
@@ -0,0 +1,51 @@
+name: Publish simulator archive
+
+on:
+  release:
+    types: [ created ]
```

You'd be better off using published here, as that means that drafting releases doesn't trigger this, and then it'll be triggered on the final publish. Creating without drafting implies publishing.

```suggestion
    types: [ published ]
```

src: https://github.com/RealOrangeOne/zoloto/blob/de61a0a2068f51a316ba25a6a435336029b6706d/.github/workflows/pypi.yml#L5